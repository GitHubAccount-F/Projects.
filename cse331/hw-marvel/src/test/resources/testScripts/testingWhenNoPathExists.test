#Tests when given a csv file, in proper format,
#but there exists no path between any of the characters

#Loads in a graph
LoadGraph graph1 noHerosInSameComicBook.csv

#Finds a path, but the path shouldn't exist
FindPath graph1 SpiderMan Hulk

#Finds another path, that shouldn't exist
FindPath graph1 SpiderMan Scarlet-Witch

#Finds another but characters don't exist in graph, so path
#shouldn't exist and message should pop out
FindPath graph1 BatMan SuperMan

#Finds another path to itself, which should exist
FindPath graph1 Wolverine Wolverine


#Tests to make sure no children are found.
ListChildren graph1 SpiderMan
ListChildren graph1 Hulk