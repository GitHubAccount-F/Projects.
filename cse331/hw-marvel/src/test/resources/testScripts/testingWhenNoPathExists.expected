#Tests when given a csv file, in proper format,
#but there exists no path between any of the characters

#Loads in a graph
loaded graph graph1

#Finds a path, but the path shouldn't exist
path from SpiderMan to Hulk:
no path found

#Finds another path, that shouldn't exist
path from SpiderMan to Scarlet-Witch:
no path found

#Finds another but characters don't exist in graph, so path
#shouldn't exist and message should pop out
unknown: BatMan
unknown: SuperMan

#Finds another path to itself, which should exist
path from Wolverine to Wolverine:


#Tests to make sure no children are found.
the children of SpiderMan in graph1 are:
the children of Hulk in graph1 are: